packages/client/src/publish/GroupKeyQueue.ts

```diff
@@ -0,0 +1,45 @@
+import { GroupKey } from '../encryption/GroupKey'
+import { GroupKeyStore } from '../encryption/GroupKeyStore'
+
+export class GroupKeyQueue {
+
+    private currentGroupKey: GroupKey | undefined // current key id if any
```

Comment:
Is code comment up-to-date? Talking about `current key id` but here we have a `GroupKey`?